### DISCUSSION TITLE
IP based surpression for all rules

### DISCUSSION THREAD

**Discussion by chrisla23** [Security-Onion-Solutions/securityonion#4903] [Unsupported Versions]
I am trying to make an alert suppression that blanket covers all rules for a given set of IPs. Reading the documentation I can see how to do it per-rule. In this case, I want to make a complete exception for an IP since they are our Qualys scanners. 

Is this possible?

Thanks,

-Chris

**Comment by TOoSmOotH** [comment]
I would just use a BPF vs modifying the rules.

**Reply by chrisla23** [reply]
I can see how I could do that to block traffic where the Qualys scanner is scanning  SO nodes, but how would I write a filter to tell it to ignore the legit pcap traffic between the scanner and other client systems? 

Good to see you, took the class a few weeks back.

Thanks,

-Chris

**Comment by joeb1kenobe** [comment]
I think the bpf filter

! src host 1.2.3.4

In each of the filter sections will cause so to ignore all traffic from
address 1.2.3 4.

Joe



On Fri, Jul 23, 2021 at 8:05 PM chrisla23 ***@***.***> wrote:

> I can see how I could do that to block traffic where the Qualys scanner is
> scanning SO nodes, but how would I write a filter to tell it to ignore the
> legit pcap traffic between the scanner and other client systems?
>
> Good to see you, took the class a few weeks back.
>
> Thanks,
>
> -Chris
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/4903#discussioncomment-1044195>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAD6W6WG74USCQVWSBZNHYTTZH7V7ANCNFSM5A22PW5Q>
> .
>
-- 
Joe Brown

**Comment by chrisla23** [comment]
Thanks,

I will try that.

-Chris
